stylesheet/index.css

```diff
@@ -0,0 +1,287 @@
+@import url(main.css);
+
+.main--banner-container {
+    display: flex;
+
+    flex-direction: column;
+
+    position: relative;
+
+    height: auto;
+    max-width: 100%;
+
+    padding-bottom: 10%;
```

Comment:
ai grija cu padding in procente ca e relative la marimea paginii .. da resize la fereastra si vezi cum arata ... incearca cu `vw` `vh` .. sau chiar `calc`

Reply:
am adaugat 10vw